tests/Fixtures/app/console

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
```

Comment:
It follows Symfony directories structure, but according to Symfony 3 it could be in a bin directory in `tests/Fixtures`